Pod/Classes/WPPHAssetDataSource.m

```diff
@@ -75,7 +75,7 @@ - (void)photoLibraryDidChange:(PHChange *)changeInstance
 
         if (!groupChangeDetails && !assetsChangeDetails && !albumChangeDetails) {
             [self.observers enumerateKeysAndObjectsUsingBlock:^(NSUUID *key, WPMediaChangesBlock block, BOOL *stop) {
```

Comment:
Im curious, why are we enumerating on the observers if there were no changes? or am I misunderstanding the code?

Reply:
Because when there is no details of the changes but the delegate was called is because you need to do a full reload. So there is still changes there but delegate is not able to send the details of it.